src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Number can not be less than 0');
+    }
+
+    const longestSide = Math.max(a, b, c);
+
+    if ((a + b + c) - longestSide <= longestSide) {
+      throw new Error('These sides can not be used to form a triangle');
+    }
+  }
+
+  getArea(): number {
+    const s = 0.5 * (this.a + this.b + this.c);
+
+    return +(Math.sqrt(
+      s * (s - this.a) * (s - this.b) * (s - this.c),
+    )).toFixed(2);
```

Comment:
You can reuse here function that you will create, instead of using `toFixed`(check my other comments)